<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>381</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>381</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Domain Details</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/icon.ico</normaloff>img/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:rgb(47, 47, 47);
color:rgb(200, 200, 200)
}
QTabWidget::pane {
	border: 1px solid rgb(128, 128, 128);
	border-radius: 3px;
}
QTabBar::tab 
{
    background: #48555E;
    color: white;   
	padding:5px;
    border: 1px solid #000;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
}
QTabBar::tab:selected, 
QTabBar::tab:hover 
{
    border-top-color: #1D2A32;
    border-color: #40494E;
    color: black;
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #C1D8E8, stop: 1 #F0F5F8); 
}
QLineEdit, QComboBox, QTextEdit{
	background-color: rgb(85,85,85);
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
	padding: 5px;
	border-radius:3px;
}
QPushButton{
	background:rgb(102, 102, 102);
	color:rgb(255, 255, 255);
	border-radius:3px;
	padding-top:5px;
	padding-bottom:5px;
	padding-right:10px;
	padding-left:10px;
	border-color: rgb(0, 0, 0);
	border-width : 1px;
	border-style:inset;
}
QPushButton:hover{
	background-color:rgb(128, 128, 128);
}
QPushButton:pressed{
	background-color:#A4A4A4;
	color:#000
}
QLabel{
	border:0px;
}
QCheckBox{
	border:0px;
}
QTextBrowser{
	color: rgb(230, 230, 230);
	padding:10px;
	background-image: url(template/img/logo.png);
	background-repeat: no-repeat;
  	background-attachment: fixed;
 	background-position: center;
}
QGroupBox {
    border: 2px solid gray;
    border-radius: 5px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 5px;
}
QGroupBox::indicator {
    width: 13px;
    height: 13px;
}
QGroupBox#groupBoxNotifications::indicator:unchecked, QGroupBox#groupBoxNotifications::indicator:checked {
	image: url(template/img/notification.png);
}
QGroupBox#groupBoxSettings::indicator:unchecked, QGroupBox#groupBoxSettings::indicator:checked {
	image: url(template/img/settings.png);
}

QGroupBox#groupBoxBrowsers::indicator:unchecked, QGroupBox#groupBoxBrowsers::indicator:checked {
	image: url(template/img/browser.png);
}

QGroupBox#groupBoxTelegram::indicator:unchecked, QGroupBox#groupBoxTelegram::indicator:checked{
	image: url(template/img/telegram.png);
}

QGroupBox#groupBoxAccountSettings::indicator:unchecked, QGroupBox#groupBoxAccountSettings::indicator:checked{
	image: url(template/img/user.png);
}

QMenuBar {
    background-color: rgb(40,40,40);
}
QMenuBar::item {
    padding: 4px 8px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: rgb(60,60,60);
	color:rgb(200,200,200);
}

QMenuBar::item:pressed {
    background: #888888;
	color: #000;
}
QHeaderView::section, QHeaderView{
	background: #48555E;
	border: 0px;
	padding-left:3px;
}
QHeaderView::item{
	border: 0px;
}
QTableCornerButton::section {
    background: #48555E;
}
QTableWidget{
	border: 1px solid gray;
	border-radius:5px;
}
QTableWidget#tableWidgetPlanetList{
	border: 2px solid gray;
	border-radius:5px;
}
</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Development Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Name servers</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Original Registrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>150</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Original DNS Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDomainName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOriginalRegistrar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOriginalDNSHost">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxDevelopmentMode">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>12</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditNameServers">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>170</y>
     <width>200</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>3</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>10</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>98</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
